openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -126,17 +130,28 @@
 
     private final String webContext;
 
-    private final OpenAPI model;
+    private OpenAPI model = null;
     private final ConcurrentMap<Format, String> cachedDocuments = new ConcurrentHashMap<>();
     private final Map<Class<?>, ExpandedTypeDescription> implsToTypes;
     private final CorsEnabledServiceHelper corsEnabledServiceHelper;
 
-    private OpenAPISupport(Builder builder) {
+    /*
+     * To handle the MP case, we must defer constructing the OpenAPI in-memory model until after the server has instantiated
+     * the Application instances. By then the builder has already been used to build the OpenAPISupport object. So save the
+     * following raw materials so we can construct the model at that later time.
+     */
+    private final OpenApiConfig openApiConfig;
+    private final OpenApiStaticFile openApiStaticFile;
+    private final Supplier<List<? extends IndexView>> indexViewsSupplier;
+
+    protected OpenAPISupport(Builder builder) {
```

Comment:
Should be `Builder<?, ?>` so we do not have a `Raw` use of class with type parameters

Reply:
Yes, this should not use the raw type. For the moment I've changed it to `Builder<T, B>` to strongly type the parameter.